src/components/tables/DataTable.vue

```diff
@@ -609,6 +610,7 @@ watch(search, () => {
               :key="subIndex"
               :class="[
                 css.td,
+                column.cellClass,
                 column.align === 'right' ? css.align__right : css.align__left,
```

Comment:
can we maybe add the center alignment too?

```
align?: 'left' | 'center' | 'right';
```

then in class
```
- column.align === 'right' ? css.align__right : css.align__left
+ column[`align__${align}`]
```

also in css
```
&.align__center {
  @apply text-center;
}

Reply:
Updated